rfc/0015-multi-file-jayvee/0015-multi-file-jayvee.md

```diff
@@ -65,120 +100,100 @@ valuetype MyValueType2 {
 publish MyValueType2;
 
 // publish later under a different name
-publish MyValueType2 called MyValueType3;
+publish MyValueType2 as MyValueType3;
 ```
 
-### Bundling elements to a library for usage elsewhere (outside of the project)
+### Packages: bundling elements to a package for decoupled usage
 
-For bundling and publishing elements, the RFC introduces a new concept called `libraries`.
-A `library` can inhibit `Valuetype`s, `Block`s, `BlockType`s, `Constraint`s, and `Transform`s.
-A library is published per default.
-All elements within a library have to use the `publish` keyword.
+For bundling and publishing elements, the RFC introduces a new concept called `packages`.
+A `package` can inhibit `Valuetype`s, `Block`s, `BlockType`s, `Constraint`s, `Transform`s, and further `Package`s.
+A package must be of visibility `file-published` and, thus, requires the keyword `publish`.
```

Comment:
This means packages in packages must be `publish`ed as well. I am happy with that.

Reply:
Should I make that explicit as implication?

Reply:
I think so, yes.